reference/docs-conceptual/Crescendo/get-started/research-tool.md

```diff
@@ -0,0 +1,141 @@
+>
+> For more information, see the [Installing the azcmagent tool](#installing-the-azcmagent-tool)
+> section of this article.
+
+## Start with command-line help and documentation
+
+Many command-line tools include a switch or parameter to display help content. Most modern
+command-line tools provide multiple levels of help for the various use-case scenarios the tool
+provides. For example, running `azcmagent` without any parameters displays the top-level help, which
+contains a list of subcommands.
+
+```output
+...
+Usage:
+  azcmagent [command]
+
+Available Commands:
+  check       Runs connectivity checks
+  config      Change configuration settings for this machine
+  connect     Connects this machine to Azure
+  disconnect  Disconnects this machine from Azure
+  help        Help about any command
+  license     Display the End-user license agreement file
+  logs        Creates a .zip file containing relevant logs. This is primarily useful for troubleshooting.
+  show        Gets machine metadata and Agent status. This is primarily useful for troubleshooting.
+  version     Display the Hybrid Management Agent version
+...
+```
+
+Each of the subcommands can have their own subcommands and parameters. For example, the `config`
+subcommand has five subcommands.
+
+```powershell
+PS> azcmagent config --help
+Change configuration settings for this machine
+
+Usage:
+  azcmagent config [command]
+
+Available Commands:
+  clear       Clear a configuration property's value
+  get         Get a configuration property's value
+  info        Describes the config properties users can set
+  list        List all configuration properties and values
+  set         Set a value for a configuration property
+
+Flags:
+  -h, --help      help for config
+      --version   version for config
+
+Global Flags:
+      --config string   config file (default is $HOME/.azcmagent.yaml)
+  -j, --json            Output in JSON format
+      --log-stderr      Redirect error and verbose messages to stderr
+  -v, --verbose         Increase logging verbosity to show all logs
+
+Use "azcmagent config [command] --help" for more information about a command.
+```
+
+Use the command-line help to discover the possible use cases. You can redirect the output of each
+help command to a file that you can use for later reference as you create your Crescendo cmdlets.
+
+> [!TIP]
+> If the help content is structured consistently, it may be possible to create code that builds
+> cmdlets by parsing this help output. Crescendo comes with some experimental help parsers to
+> illustrate how this may be accomplished. See the `Experimental` folder in the root folder of the
+> **Microsoft.PowerShell.Crescendo** module.
+
+Take note of the output formats that the command-line tool offers. Many command-line tools can
+output information in formats such as CSV or JSON. These structured formats are easily converted to
+PowerShell objects.
```

Comment:
Nit: Is it worth explicitly mentioning `ConvertFrom-Json` and `ConvertFrom-Csv` here? I'm not _sure_ it is.